src/applications/ivc-champva/10-7959C/chapters/healthInsuranceInformation.js

```diff
@@ -110,133 +113,168 @@ export function applicantInsuranceEffectiveDateSchema(isPrimary) {
   };
 }
 
-export const applicantPrimaryExpirationDateSchema = {
-  uiSchema: {
-    applicants: {
-      'ui:options': {
-        viewField: ApplicantField,
-      },
-      items: {
-        ...titleUI(
-          ({ formData }) =>
-            `${applicantWording(formData)} ${
-              formData?.applicantPrimaryProvider
-            } insurance expiration date`,
-        ),
-        applicantPrimaryExpirationDate: currentOrPastDateUI(
-          'Health insurance expiration date',
-        ),
+export function applicantInsuranceExpirationDateSchema(isPrimary) {
+  const keyname = isPrimary
+    ? 'applicantPrimaryExpirationDate'
+    : 'applicantSecondaryExpirationDate';
+  return {
+    uiSchema: {
+      applicants: {
+        'ui:options': { viewField: ApplicantField },
+        items: {
+          ...titleUI(
+            ({ formData }) =>
+              `${applicantWording(formData)} ${insuranceName(
+                formData,
+                isPrimary,
+              )} ${
+                isPrimary ? 'primary' : 'secondary'
+              } insurance expiration date`,
+          ),
+          [keyname]: currentOrPastDateUI('Health insurance expiration date'),
+        },
```

Comment:
The majority of this PR is this sort of adjustment. I modified the schema object for each of the primary insurance pages so that it can work for the secondary insurance page as well, based off of the boolean passed in.